src/chrome/content/rules/Stack-Exchange.xml

```diff
@@ -48,6 +48,14 @@
 		<exclusion pattern="^http://blog\.gaming\.stackexchange\.com"/>
 		<test url="http://blog.gaming.stackexchange.com/" />
 
+		<!-- this causes login issues on area51. See:
+			- https://area51.meta.stackexchange.com/questions/13597/area-51-openid-login-is-broken-with-https/27726
```

Comment:
Could you also add https://meta.stackexchange.com/questions/292058/network-wide-https-its-time/306308#306308?